src/main/java/rs117/hd/HdPlugin.java

```diff
@@ -2188,11 +2184,12 @@ public void onConfigChanged(ConfigChanged event)
 				recompileProgram();
 				break;
 			case "unlockFps":
-				configUnlockFps = config.unlockFps();
+			case "vsyncMode":
+			case "fpsTarget":
 				clientThread.invokeLater(() ->
```

Comment:
This invokeLater is unnecessary

Reply:
Right you are 🙌 Thanks for the feedback. I've addressed that here: https://github.com/RS117/RLHD/pull/181/commits/e1f3b4a4efd0e782fdee2ae42ee6e036cfbf3737